File: java/src/test/java/shapes/Fixtures.java
package shapes;

import shapes.base.models.Circle;
import shapes.base.models.Rectangle;
import shapes.base.models.Square;

class Fixtures {
    static Circle setupCircle() {
        double radius = 15.5;
        return new Circle(radius);
    }

    static Rectangle setupRectangle() {
        double width = 22.75;
        double height = 33.25;
        return new Rectangle(width, height);
    }

    static Square setupSquare() {
        double side = 10.25;
        return new Square(side);
    }
}

File: php/tests/Fixtures/Fixtures.php
<?php

namespace Shapes\Tests\Fixtures;

use Shapes\Base\Models\{Circle, Rectangle, Square};

function setup_circle(): Circle {
    $radius = 15.5;
    return new Circle($radius);
}

function setup_rectangle(): Rectangle {
    $length = 22.75;
    $width = 33.25;
    return new Rectangle($length, $width);
}

function setup_square(): Square {
    $side = 10.25;
    return new Square($side);
}
